Zilon.Core/Zilon.Core/PersonModules/CombatActModule.cs

```diff
@@ -7,13 +20,206 @@ namespace Zilon.Core.PersonModules
     /// </summary>
     public sealed class CombatActModule : ICombatActModule
     {
-        public CombatActModule()
+        private readonly ITacticalActScheme _defaultActScheme;
+        private readonly IEquipmentModule _equipmentModule;
+        private readonly IEffectsModule _effectsModule;
+        private readonly IEvolutionModule _evolutionModule;
+
+        public CombatActModule(
+            ITacticalActScheme defaultActScheme,
+            IEquipmentModule equipmentModule,
+            IEffectsModule effectsModule,
+            IEvolutionModule evolutionModule)
         {
             IsActive = true;
+
+            _defaultActScheme = defaultActScheme;
+            _equipmentModule = equipmentModule;
+            _effectsModule = effectsModule;
+            _evolutionModule = evolutionModule;
         }
 
         public ITacticalAct[] Acts { get; set; }
         public string Key { get => nameof(ICombatActModule); }
         public bool IsActive { get; set; }
+
+        public IEnumerable<ITacticalAct> CalcCombatActs()
+        {
+            var perks = GetPerksSafe();
+            return CalcActs(_defaultActScheme, _equipmentModule, _effectsModule, perks);
+        }
+
+        private static ITacticalAct[] CalcActs(ITacticalActScheme defaultActScheme,
+            IEnumerable<Equipment> equipments,
+            IEffectsModule effects,
+            IEnumerable<IPerk> perks)
+        {
+            if (equipments is null)
+            {
+                throw new ArgumentNullException(nameof(equipments));
+            }
+
+            var actList = new List<ITacticalAct>();
+
+            var defaultAct = CreateTacticalAct(defaultActScheme, equipment: null, effects: effects, perks: perks);
+            actList.Insert(0, defaultAct);
+
+            if (equipments != null)
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Change this condition so that it does not always evaluate to 'true'.](https://app.codacy.com/manual/kreghek/Zilon_Roguelike/pullRequest?prid=5480188)